### DISCUSSION TITLE
Set the completion key to the right arrow key

### DISCUSSION THREAD

**Discussion by treymerkley** [company-mode/company-mode#1199] [Q&A] [answered]
Related to #640, I'd like to set up Company to only complete when I press the right arrow key instead of RET as it currently stands. How would I go about setting that up? Is there a way to do it in one binding or do I bind completion to the right arrow and unbind to RET separately?

**Comment by dgutov** [comment, marked as answer]
> Is there a way to do it in one binding

Nope.

> do I bind completion to the right arrow and unbind to RET separately

Yep.
